retrieval/series_cache.go

```diff
@@ -337,6 +337,9 @@ func (c *seriesCache) refresh(ctx context.Context, ref uint64) error {
 		return nil
 	}
 	// Remove target labels and __name__ label.
+	// Stackdriver only accepts a limited amount of labels, so we choose to economize aggressively here. This should be OK
```

Comment:
What do you think about adding quota reference (https://cloud.google.com/monitoring/quotas) next to 'limited amount of labels', so readers of the code will recognize the limitation?

Reply:
I think it's a fine idea, but I don't think it adds valuable information to help understand the code.